src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -391,13 +400,33 @@ public ArticlePageObject clickPublishButton() {
     return new ArticlePageObject(driver);
   }
 
-  public void addCategoryToSourceCode(String catName) {
-    sourceModeTextArea.sendKeys(catName);
-  }
-
   public TemplatePageObject clickPublishButtonInTemplateNamespace() {
     wait.forElementVisible(submitButton);
     submitButton.click();
     return new TemplatePageObject(driver);
   }
+
+  public SourceEditModePageObject addContentToTextArea(String text) {
+    wait.forElementVisible(textAreaScroller);
+    new Actions(driver)
+        .click()
+        .sendKeys(text)
+        .build().perform();
+    return this;
+  }
+
+  public SourceEditModePageObject deleteCharacter(Integer howMany) {
```

Comment:
howMany should be renamed to characterCount, as a variable that stores values should be a precise noun, not a question or a sentence